pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -521,3 +531,79 @@ func checkForPodSecondaryNetworkAttachement(pod *corev1.Pod) (string, bool) {
 		return netObj, false
 	}
 }
+
+// initializeSecondaryInterfaceStore restores secondary interfaceStore when agent restarts.
+func (pc *PodController) initializeSecondaryInterfaceStore() error {
+	if pc.ovsBridgeClient == nil {
```

Comment:
You meant this can happen in a non-UT agent run? How can? pc.ovsBridgeClient was already set before initializeSecondaryInterfaceStore()

Reply:
As we talked offline, this can happen when secondary bridge is not configured, meaning no VLAN interfaces at all. In this case, we should skip both initializeSecondaryInterfaceStore and reconcileSecondaryInterfaces. And I suggest doing this in NewPodController()